docset/mdop/appvclient/add-appvclientpackage.md

```diff
@@ -113,11 +113,12 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 [Remove-AppvClientPackage](./Remove-AppvClientPackage.md)
 
 [Repair-AppvClientPackage](./Repair-AppvClientPackage.md)
-
+./get-appvclientpackage.md
 [Set-AppvClientPackage](./Set-AppvClientPackage.md)
-
+./mount-appvclientpackage.md
 [Stop-AppvClientPackage](./Stop-AppvClientPackage.md)
-
+./publish-appvclientpackage.md
 [Unpublish-AppvClientPackage](./Unpublish-AppvClientPackage.md)
+./publish-appvclientpackage.md
 
-
+./remove-appvclientpackage.md./repair-appvclientpackage.md./set-appvclientpackage.md./stop-appvclientpackage.md./unpublish-appvclientpackage.md
```

Comment:
is there a reason why these are unformatted links?

Reply:
Its messed up. I will fix it.